### PR TITLE
Updated lasagna recipe

### PR DESCRIPTION
**Pull request by yiusa**
expanded the existing lasagna recipe

### CODE REVIEW

File: src/ricotta-lasagna-filling.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 0 min
+- 🍽️ Servings: 5-6
+
+## Ingredients
+
+- Ricotta cheese 
+- One large egg
+- Parmesan cheese
+- oregano (optional)
```
Comment (Socerest2) [inline src/ricotta-lasagna-filling.md:14]:
```suggestion
- Oregano (Optional)
```
- Minor Grammar Fix

File: src/ricotta-lasagna-filling.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Directions
+
+1. mix the ricotta cheese together with a handfull of finely grated parmesan.
```
Comment (Socerest2) [inline src/ricotta-lasagna-filling.md:18]:
```suggestion
1. Mix the ricotta cheese together with a handfull of finely grated parmesan.
```

File: src/ricotta-lasagna-filling.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+2. add the oregano and the egg and mix.
```
Comment (Socerest2) [inline src/ricotta-lasagna-filling.md:19]:
```suggestion
2. Add the oregano and the egg and mix.
```

File: src/lasagna.md [hunk 4]
```diff
@@ -1,34 +1,38 @@
...
+2. When the sauce is almost done cooking prepare your béchamel or ricotta filling.
+3. If you want you can add 3-4 handfulls of grated emental cheese to the béchamel.
+4. If necessary, boil the lasagna noodles following the instructions on the packaging, start preheating your oven to 175°C/350°F.
+5. Spread a thin layer of pasta sauce on the bottom of a large, deep baking tray.
+6. Cover with lasagna sheets and spread a layer of béchamel sauce or ricotta, cover with lasagna sheets and spread a layer of pasta sauce if you want you can finely grate a little bit of parmesan on each layer of bolognese.
+7. Rinse and repeat the previous step until the tray is nearly full or you are running low on béchamel sauce/ricotta filling, take care that the last layer should be bechamel or ricotta.
+8. Grate some parmesan on top of your final layer and cover with Dry mozarella cheese, wrap the top in aluminum foil.
+9. Place the tray in the centre of your pre-heated oven and let it cook for 30-35 minutes, after that remove the foil to let the top brown.
+10. once it is done cooking take it out of the oven and cover again with aluminum foil, let it rest for 10 minutes before serving.
```
Comment (Socerest2) [inline src/lasagna.md:31]:
```suggestion
10. Once it is done cooking take it out of the oven and cover again with aluminum foil, let it rest for 10 minutes before serving.
```

**Review by Socerest2 — CHANGES_REQUESTED** [review]
Just a small bit of grammar fixes

**Comment by yiusa** [comment]
> Just a small bit of grammar fixes

thanks for the check should be fixed.

**Review by Socerest2 — APPROVED** [review]
Looks Good!

**Review by hallzy — APPROVED** [review]

**Comment by LukeSmithxyz** [comment]
Get rid of the meat tag.

**Comment by LukeSmithxyz** [comment]
There's also no reason this needs to be named "Fresh" Lasagna.

File: src/lasagna.md [hunk 5]
```diff
@@ -1,34 +1,38 @@
-# Lasagna
+# Fresh lasagna
 
-A classic italian dish, lasagna is a great meal for a cold day when you're in the mood for pasta. 
+![fresh-lasagna](pix/fresh-lasagna.webp)
```
Comment (Socerest2) [inline src/lasagna.md:3]:
```suggestion
![Lasagna](pix/fresh-lasagna.webp)
```
Lukes Stuff

File: src/lasagna.md [hunk 6]
```diff
@@ -1,34 +1,38 @@
-# Lasagna
+# Fresh lasagna
```
Comment (Socerest2) [inline src/lasagna.md:1]:
```suggestion
# Lasagna
```
Per Luke Request

File: src/lasagna.md [hunk 7]
```diff
@@ -1,34 +1,38 @@
...
+9. Place the tray in the centre of your pre-heated oven and let it cook for 30-35 minutes, after that remove the foil to let the top brown.
+10. Once it is done cooking take it out of the oven and cover again with aluminum foil, let it rest for 10 minutes before serving.
 
 ## Contribution
 
+- Yiusa, eth `0x68f1317c6512f0267fa711cafb6c134ae968fa80`
 - Josiah -- [website](https://himiko.cloud), [donate](https://himiko.cloud/donate/)
 
-;tags: basic italian
+;tags: italian pasta meat
```
Comment (Socerest2) [inline src/lasagna.md:38]:
```suggestion
;tags: italian pasta
```

**Review by Socerest2 — CHANGES_REQUESTED** [review]
Github glitch it got posted twice. Anyways. heres some stuff to fix based on lukes comments.

File: src/lasagna.md [hunk 8]
```diff
@@ -1,34 +1,38 @@
...
-- Nine or ten oven ready lasagna noodles.
-- Medium sized jar of tomato/pasta sauce.
-- Medium sized bag of mozzarella cheese.
-- Small bag of parmesan cheese.  
-- Fresh basil, dried oregano, for taste.
+
+- Parmesan cheese
+- Emental cheese (optional)
+- Dry mozarella cheese
+- [fresh Bolognese sauce](bechamel-sauce.html) or about 1 litre worth of store-bought sauce
```
Comment (printplaatreparatie) [inline src/lasagna.md:16]:
```suggestion
- [Fresh Bolognese sauce](bolognese-sauce.html) or about 1 litre worth of store-bought sauce
```

**Comment by yiusa** [comment]
Alright, fixed the recipes like you guys suggested. Should be all good now. Thanks Socerest2 and printplaatreparatie for the 
proofreading.

**Comment by LukeSmithxyz** [comment]
The image name should match the markdown filename (no fresh).

**Comment by yiusa** [comment]
> The image name should match the markdown filename (no fresh).

Alright i think all should be like it's supposed to be now, ill be more careful when proofreading  my recipes before creating pull requests in the future so that this doesn't happen again.

**Pull request merged**